lib/msal-node/src/network/HttpClient.ts

```diff
@@ -107,7 +107,7 @@ const networkRequestViaProxy = <T>(
             if (statusCode < 200 || statusCode > 299) {
                 request.destroy();
                 socket.destroy();
-                reject(new Error(` Error connecting to proxy: ${response.statusCode}, ${response?.statusMessage}`));
+                reject(new Error(`Error connecting to proxy. Status Code: ${response.statusCode}. Status Message: ${response?.statusMessage || "Unknown"}`));
```

Comment:
nit: `proxy.StatusCode` instead of `proxy. Status Code`?

Reply:
The "." there represents the end of a sentence, not object.method :)

Reply:
Maybe use "Error connecting to proxy. Http status code: ..."

Reply:
:-D. I meant can we have it more like code:<Code> but your suggestion is fine too.